airbyte/assets/dashboards/airbyte_overview.json

```diff
@@ -0,0 +1,1354 @@
+          "x": 0,
+          "y": 1,
+          "width": 6,
+          "height": 3
+        }
+      }]
+    },
+    "layout": {
+      "x": 6,
+      "y": 0,
+      "width": 6,
+      "height": 6
+    }
+  }, {
+    "id": 6518790764913386,
+    "definition": {
+      "title": "Jobs Snapshot",
+      "background_color": "vivid_purple",
+      "show_title": true,
+      "type": "group",
+      "layout_type": "ordered",
+      "widgets": [{
+        "id": 2154332464605546,
+        "definition": {
+          "type": "note",
+          "content": "Metrics like number of persisting jobs, running jobs, oldest jobs, and succeeded jobs are useful for monitoring your Airbyte instance because they provide insights into the platform's performance, job processing efficiency, and data synchronization success to ensure smooth data integration workflows. ",
+          "background_color": "purple",
+          "font_size": "14",
+          "text_align": "center",
+          "vertical_align": "center",
+          "show_tick": false,
+          "tick_pos": "50%",
+          "tick_edge": "left",
+          "has_padding": true
+        },
+        "layout": {
+          "x": 0,
+          "y": 0,
+          "width": 12,
+          "height": 1
+        }
+      }, {
+        "id": 6329913229296406,
+        "definition": {
+          "title": "Number of jobs run unusually long",
+          "title_size": "16",
+          "title_align": "left",
+          "type": "query_value",
+          "requests": [{
+            "formulas": [{
+              "formula": "query1"
+            }],
+            "conditional_formats": [{
+              "palette": "white_on_red",
+              "value": 15,
+              "comparator": ">="
+            }, {
+              "palette": "green_on_white",
+              "value": 15,
+              "comparator": "<"
+            }],
+            "response_format": "scalar",
+            "queries": [{
+              "query": "avg:airbyte.metrics_reporter.num_unusually_long_syncs{*}.rollup(max, 1800)",
+              "data_source": "metrics",
+              "name": "query1",
+              "aggregator": "last"
+            }]
+          }],
+          "autoscale": true,
+          "precision": 2,
+          "timeseries_background": {
+            "type": "area",
+            "yaxis": {
+              "include_zero": false
+            }
+          }
+        },
+        "layout": {
+          "x": 0,
+          "y": 1,
+          "width": 6,
+          "height": 2
+        }
+      }, {
+        "id": 4596888485669944,
+        "definition": {
+          "title": "Jobs Succeeded",
+          "title_size": "16",
+          "title_align": "left",
+          "type": "query_value",
+          "requests": [{
+            "response_format": "scalar",
+            "queries": [{
+              "name": "query1",
+              "data_source": "metrics",
+              "query": "sum:airbyte.worker.job_succeeded_by_release_stage{*}.as_count()",
```

Comment:
@iliakur I'm not sure where to report this, but this dashboard does not work because the charts use `airbyte.worker.job_xxx` pattern for metrics but configuration in dogstatsd_mapper_profiles changes the metrics e.g. from `worker.job_*` to  `airbyte.worker.job.$1` (replacing underscore with dot) for all metrics.

Reply:
@jmisur Thanks for the report! I'm fixing this and some other problems I found with the dashboard. I'll ping you when it's ready to be reviewed.